api/diagnosis/diagnosis_routes.py

```
@@ -0,0 +1,88 @@
+@router.get("/api/diagnosis/drowiness")
+async def get_diagnosis_result(request: Request, device_uid: str = Query(..., alias="deviceUid")):
+    model = request.app.state.model
+
+    if model is None:
+        logger.error("error_code:500, model_not_loaded, get_diagnosis_result(모델 로드 확인), 모델이 로드되지 않음")
+        return create_error_response(500, "model_not_loaded", "get_diagnosis_result(모델 로드 확인)", "모델이 로드되지 않음")
+
+    try:
+        frames = await get_frames_from_queue(device_uid)
+        if not frames:
+            raise ErrorForm(404, "no_frames", "get_diagnosis_result(프레임 조회)")
```

Ruff (0.11.9)

Abstract `raise` to an inner function

(TRY301)